Makefile

```diff
@@ -21,7 +22,7 @@ DOCKER_ENVS := \
 	-e DOCKER_EXPERIMENTAL \
 	-e DOCKER_GITCOMMIT \
 	-e DOCKER_GRAPHDRIVER=$(DOCKER_GRAPHDRIVER) \
-	-e DOCKER_INCREMENTAL_BINARY \
+	-e DOCKER_INCREMENTAL_BINARY=$(DOCKER_INCREMENTAL_BINARY) \
```

Comment:
actually it's not equivalent because we *set* `DOCKER_INCREMENTAL_BINARY` (and same goes for `DOCKER_GRAPHDRIVER`) in the `Makefile` – and thus the `docker run` command doesn't pick it up :sweat: